tests/test_settings_api.py

```diff
@@ -21,77 +21,61 @@ def test_setup_models_viscous_model_settings(new_solver_session) -> None:
 def test_wildcard(new_solver_session):
     solver = new_solver_session
     case_path = download_file("elbow_source_terms.cas.h5", "pyfluent/mixing_elbow")
     solver.file.read_case(file_name=case_path)
     boundary_conditions = solver.setup.boundary_conditions
-    assert boundary_conditions.velocity_inlet["inl*"].vmag() == {
-        "inlet2": {"vmag": {"option": "value", "value": 15}},
-        "inlet1": {"vmag": {"option": "value", "value": 5}},
+    assert boundary_conditions.velocity_inlet["inl*"].momentum.velocity() == {
+        "inlet2": {"momentum": {"velocity": {"option": "value", "value": 15}}},
+        "inlet1": {"momentum": {"velocity": {"option": "value", "value": 5}}},
     }
-    assert boundary_conditions.velocity_inlet["inl*"].vmag.value() == {
-        "inlet2": {"vmag": {"value": 15}},
-        "inlet1": {"vmag": {"value": 5}},
+    assert boundary_conditions.velocity_inlet["inl*"].momentum.velocity.value() == {
+        "inlet2": {"momentum": {"velocity": {"value": 15}}},
+        "inlet1": {"momentum": {"velocity": {"value": 5}}},
     }
-    boundary_conditions.velocity_inlet["inl*"].vmag = 10
-    assert boundary_conditions.velocity_inlet["inl*"].vmag() == {
-        "inlet2": {"vmag": {"option": "value", "value": 10}},
-        "inlet1": {"vmag": {"option": "value", "value": 10}},
+    boundary_conditions.velocity_inlet["inl*"].momentum.velocity = 10
+    assert boundary_conditions.velocity_inlet["inl*"].momentum.velocity() == {
+        "inlet2": {"momentum": {"velocity": {"option": "value", "value": 10}}},
+        "inlet1": {"momentum": {"velocity": {"option": "value", "value": 10}}},
     }
     boundary_conditions.velocity_inlet = boundary_conditions.velocity_inlet[
         "inl*"
-    ].vmag()
-    assert boundary_conditions.velocity_inlet["inl*"].vmag() == {
-        "inlet2": {"vmag": {"option": "value", "value": 10}},
-        "inlet1": {"vmag": {"option": "value", "value": 10}},
+    ].momentum.velocity()
+    assert boundary_conditions.velocity_inlet["inl*"].momentum.velocity() == {
+        "inlet2": {"momentum": {"velocity": {"option": "value", "value": 10}}},
+        "inlet1": {"momentum": {"velocity": {"option": "value", "value": 10}}},
     }
-    assert boundary_conditions.fluid["*"].source_terms["*mom*"]() == {
+    cell_zone_conditions = solver.setup.cell_zone_conditions
+    assert cell_zone_conditions.fluid["*"].source_terms.source_terms["*mom*"]() == {
         "fluid": {
             "source_terms": {
-                "y-momentum": [{"option": "value", "value": 2}],
-                "x-momentum": [{"option": "value", "value": 1}],
-                "z-momentum": [{"option": "value", "value": 3}],
+                "source_terms": {
+                    "x-momentum": [{"option": "value", "value": 1}],
+                    "y-momentum": [{"option": "value", "value": 2}],
+                    "z-momentum": [{"option": "value", "value": 3}],
+                }
```

Comment:
This repetition seems like it could be improved, in `cell_zone_conditions` of `source_terms.source_terms` and `["source_terms"]["source_terms"]` for the dict 

Though I don't know how much trouble it would be to change this